unit/util/sharing_map.cpp

```diff
@@ -6,11 +6,12 @@ Author: Daniel Poetzl
 
 \*******************************************************************/
 
-#define SHARING_MAP_INTERNAL_CHECKS
-#define SHARING_NODE_INTERNAL_CHECKS
+#define SM_INTERNAL_CHECKS
+#define SN_INTERNAL_CHECKS
 
 #include <climits>
 #include <random>
+#include <set>
```

Comment:
This seems unrelated to the topic of the PR?

Reply:
The newly introduced functions `count_unmarked_nodes()` and `get_sharing_stats()` use `std::set` which was not used before in the sharing map.